cloud-functions/Manual-Steps.md

```diff
@@ -0,0 +1,388 @@
+# Manual Steps for this sample
```

Comment:
why two different markdown files?

Reply:
Because I wanted to make it really clear, that there is a choice, and if the person chooses the cloudshell tutorial route, they don't need to follow any further steps in the main readme.

Reply:
Its fine. I have merged it..